addons/mail/static/src/components/notification_list/tests/notification_list_notification_group_tests.js

```diff
@@ -98,42 +98,33 @@ QUnit.test('notification group basic layout', async function (assert) {
 });
 
 QUnit.test('mark as read', async function (assert) {
-    assert.expect(6);
+    assert.expect(3);
 
     // message that is expected to have a failure
     this.data['mail.message'].records.push({
         id: 11, // random unique id, will be used to link failure to message
         message_type: 'email', // message must be email (goal of the test)
-        model: 'mail.channel', // expected value to link message to channel
+        model: 'sale.order', // expected value to link message to sale.order
```

Comment:
`sale.order` should not be used in the `mail` module because it is not in the dependency chain.

If channel isn't appropriate, you can pick for example partner instead.

Reply:
Changed. On the server, we need a model that inherit from mail_thread but for the test it doesn't matter and so I have used mail.channel. (the mock server accept any model while the real server would work only on model that inherits from mail_thread (or has the same exposed method that we need here))